packages/synapse-react-client/src/components/UserCard/UserCardMedium.tsx

```diff
@@ -176,14 +168,14 @@ export function UserCardMedium({
           )}
         </p>
         {(position || company) && (
-          <p className={`${isLarge ? 'SRC-whiteText' : ''}`}>
+          <p className={`${isLarge ? 'SRC-blackText' : ''}`}>
```

Comment:
I think the default color is fine (that's what's used in the design) for both medium and large variants, so you can remove the className prop entirely for all of these where you swapped `whiteText` for `blackText`
```suggestion
          <p>
```